tests/functional/ui/test_health_overview.py

```
@@ -600,3 +701,128 @@ def test_health_score_with_blackbox_and_metrics_exporter_shutdown(
+        df_overview = page_nav.nav_storage_data_foundation_overview_page()
+        infra_health_overview = df_overview.nav_health_view_checks()
+        page_nav.take_screenshot("before_unsilence_indefinite")
+        infra_health_overview.unsilence_alert_by_type_indefinite()
+        self.alerts_silenced = False
+        logger.info("Successfully unsilenced all indefinitely silenced alerts")
+        logger.info("Waiting for alerts to trigger")
+        time.sleep(30)
+        logger.info("Verifying health score reflects active alerts after unsilencing")
+        self.update_alert_map(threading_lock)
+        expected_score_drop = 0
+        for alert in ALERT_MAP.keys():
```

Ruff (0.15.15)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)